python/feathub/processors/flink/table_builder/tests/sliding_window_test_base.py

```diff
@@ -0,0 +1,45 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import datetime
+from typing import Dict
+
+from feathub.processors.flink.table_builder.tests.table_builder_test_base import (
+    FlinkTableBuilderTestBase,
+)
+
+
+class SlidingWindowTestBase(FlinkTableBuilderTestBase):
+    def setUp(self):
+        super().setUp()
+        self.props = self.get_props()
+
+    @staticmethod
+    def _to_epoch_millis(
```

Comment:
These two static methods seem pretty general.

Would it be useful to move these methods to some util classes so that they can be used by other tests in the future?